DEA_products/DEA_Wetlands_Insight_Tool.ipynb

```diff
@@ -0,0 +1,2403 @@
+{
```

Comment:
<p>Can we add a little more background to TCI here? Like what it is intended to do, roughly how it works? It's a pretty important part of this analysis, so would be good to have a nice introduction to it here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>